src/sprinkles.css.ts

```diff
@@ -13,7 +17,7 @@ const colorProperties = defineProperties({
     },
 });
 
-const sizes = [0, 2, 4, 8, 12, 16, 24, 32, 40, 48, 56, 64, 72, 80] as const;
+const sizes = [0, 2, 4, 6, 8, 12, 16, 24, 32, 40, 48, 56, 64, 72, 80] as const;
```

Comment:
hmmm... where do you need 6px?

Reply:
paddingY: PADDING_Y_LINK,

in link stlyes

Reply:
what about the 12 value? I added this because I saw that 12 (that maybe we should remove it)